sunderland/patterns/home.php

```diff
@@ -6,8 +6,8 @@
  * Inserter: no
  */
 ?>
-<!-- wp:cover {"url":"<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/summertime_image.webp","dimRatio":10,"overlayColor":"base","minHeight":100,"minHeightUnit":"vh","style":{"spacing":{"padding":{"right":"5vw","left":"5vw"}}},"textColor":"contrast","layout":{"type":"constrained"}} -->
-<div class="wp-block-cover has-contrast-color has-text-color" style="padding-right:5vw;padding-left:5vw;min-height:100vh"><span aria-hidden="true" class="wp-block-cover__background has-base-background-color has-background-dim-10 has-background-dim"></span><img class="wp-block-cover__image-background" alt="" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/summertime_image.webp" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:group {"align":"full","style":{"dimensions":{"minHeight":"100vh"},"spacing":{"padding":{"right":"0vw","left":"0vw"}}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch","verticalAlignment":"space-between"}} -->
+<!-- wp:cover {"url":"https://playground.wordpress.net/scope:0.1783832307829636/wp-content/themes/sunderland/assets/images/summertime_image.webp","dimRatio":10,"overlayColor":"base","isUserOverlayColor":true,"minHeight":100,"minHeightUnit":"vh","lock":{"move":false,"remove":true},"style":{"spacing":{"padding":{"right":"5vw","left":"5vw"}}},"textColor":"contrast","layout":{"type":"default"}} -->
```

Comment:
This template shouldn't be updated. When I was editing the theme, the background photo disappeared, likely due to the localisation of the photo but the previous code was working fine.

Reply:
I've updated the image path so it's relative to the theme again, and I can see the cover image locally.